packages/terra-horizontal-divider/tests/nightwatch/horizontal-divider-spec.js

```diff
@@ -0,0 +1,19 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
These nightwatch tests need to be updated for the terra-toolkit v2. See [here](https://github.com/cerner/terra-core/blob/master/packages/terra-button/tests/nightwatch/button-spec.js#L1) for an example.

Reply:
Updated  https://github.com/cerner/terra-core/pull/962/commits/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-896d228d46a686daac552420c131eabfR2